scripts/README.md

```diff
@@ -0,0 +1,19 @@
+Launching scripts via the container
+===================================
+
+Assuming you have Docker privileges on deployment machines and the right tag
+for the container get this container like shown below:
+```
+docker pull qserv/dax_data_generator:tools
+```
+
+To start the client (replace 'tools' with 'tools-DM-<branch#>'):
+```
+docker run --network=host --rm -it -u 1000:1000 qserv/dax_data_generator:tools \
+  /bin/bash -c "/home/qserv/dax_data_generator/scripts/run_datagenclient.sh"
+```
+
```

Comment:
datagenclient.py only gets host and port of the server from command line arguments. All of the other configuration values are provided by datagenserver.py and sent over the network connection.

The server doesn't really need to run in a container, but it can be. Since there really should be only one datagenserver in the system it's easier to set it up outside of a container. There are a couple of server arguments that are most easily handled on the command line, like don't try to register the schema with ingest since it has already been done is '-s' or '-k' to skip ingest entirely. What you propose makes sense for the server in a container, but we don't need that at this point.

Reply:
I missed this fact (that clients don't require any external configuration). However, you the proposed  Docker's volume mounting option will be required to run the server.

I doubt your Python code would run outside of the Docker environment since it requires LSST Stack environment and Python 3. None of the target machines (Qserv clusters at NCSA) where the code is meant to be run has such environment.

Reply:
I'm adding the instruction. The config files are complex, so it may be easier to setup the stack and everything on one computer to get the configuration files correct. I suspect I'll probably try to get configurations working locally for a dozen chunks before trying to set them up on a cluster.